standard/ERCs/erc-6900.md

```diff
@@ -538,36 +527,51 @@ Finally, the function MUST emit the event `ModuleUninstalled` with the module's
 
 > **⚠️ Incorrectly uninstalled modules can prevent uninstalls of their dependencies. Therefore, some form of validation that the uninstall step completely and correctly removes the module and its usage of dependencies is required.**
 
-#### Calls to `validateUserOp`
+### Validation Call Flow
+
+Modular accounts support three different calls flows for validation: user op validation, runtime validation, and signature validation. User op validation happens within the account's implementation of the function `validateUserOp`, defined in the ERC-4337 interface `IAccount`. Runtime validation happens through the dispatcher function `executeWithAuthorization`, or when using direct call validation. Signature validation happens within the account's implementation of the function `isValidSignature`, defined in ERC-1271.
+
+For each of these validation types, an account implementation may specify its own format for selecting which validation function to use, as well as any per-hook data for validation hooks.
```

Comment:
```suggestion
For each of these validation types, an account implementation may specify its own format for selecting which validation function to use, as well as any per-hook data for validation hooks.
```

Does this conflict with SMA? It's technically still a validation function, but the connotation may be off. I think it's fine to refer to fallback validation as a validation function, wdyt?

Reply:
Good point. I think we just need to do a scan to make sure we don't enforce validation function being installed only, we should be good then. (I don't think we do)

Reply:
Good stuff. Agreed!

Reply:
Yeah, I think it's ok? This line is mostly regarding the fact that it's not standardized to pick via `userOp.signature`, but other fields like `userOp.nonce` are acceptable, too.